src/foam/nanos/auth/LanguageChoiceView.js

```diff
@@ -30,87 +30,85 @@ foam.CLASS({
   exports: [ 'as data' ],
 
   css: `
-  ^carrot {
-    width: 0;
-    height: 0;
-    border-left: 5px solid transparent;
-    border-right: 5px solid transparent;
-    border-top: 5px solid white;
-    display: inline-block;
-    float: right;
-    margin-top: 7px;
-    margin-left: 7px;
-  }
-  ^ .foam-u2-ActionView-languageChoice {
-    display: inline-block;
-    background: none !important;
-    border: 0 !important;
-    box-shadow: none !important;
-    width: max-content;
-    cursor: pointer;
-    margin-right: 27px;
-  }
-  ^ .foam-nanos-u2-navigation-TopNavigation-LanguageChoiceView {
-    align-items: center;
-  }
-  ^ .foam-u2-ActionView-languageChoice > span {
-    font-family: /*%FONT1%*/ Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
-    font-size: 16px;
-    font-weight: 300;
-    letter-spacing: 0.2px;
-    color: #ffffff;
-  }
-  ^ .popUpDropDown > div {
-    margin-left: -16px;
-    margin-right: -16px;
-    width: 100%;
-    height: 25;
-    padding: 0px 16px 5px 16px;
-    font-size: 14px;
-    font-weight: 300;
-    letter-spacing: 0.2px;
-    color: /*%BLACK%*/ #1e1f21;
-    line-height: 30px;
-  }
-  ^ .foam-u2-PopupView {
-    border-radius: 4px;
-    left: -30 !important;
-    top: 51px !important;
-    padding: 0px 16px 0px 16px !important;
-    z-index: 1000;
-    width: fit-content !important;
-    background: white;
-    opacity: 1;
-    box-shadow: 2px 2px 2px 2px rgba(0, 0, 0, 0.19);
-  }
-  ^ .popUpDropDown > div:hover{
-    background-color: #1cc2b7;
-    color: white;
-    cursor: pointer;
-  }
-  ^ .popUpDropDown::before {
-    content: ' ';
-    position: absolute;
-    height: 0;
-    width: 0;
-    border: 8px solid transparent;
-    border-bottom-color: white;
-    -ms-transform: translate(110px, -16px);
-    transform: translate(50px, -16px);
-  }
-  ^ .flag {
-    width: 30px !important;
-    height: 17.6px;
-    object-fit: contain;
-    padding-top: 6px;
-    padding-left: 10px;
-    margin-right: 23px;
-  }
-  ^ img {
-    height: 17.6px !important;
-    margin-right: 6;
-    width: auto;
-  }
+    ^container {
```

Comment:
@minsunny Why so much CSS for this menu?  Do we have custom CSS for each of the menus - left, settings, language? 
@kgrgreer @kennyqiyenkan